src/js/utils/browser.js

```diff
@@ -70,7 +68,6 @@ export const CHROME_VERSION = (function() {
   }
   return null;
 }());
-export const IS_IE8 = (/MSIE\s8\.0/).test(USER_AGENT);
```

Comment:
I am not sure if we should remove this code

Reply:
yeah, maybe we should keep these, not sure.

Reply:
Actually, I think I'm inclined to leave it out since we don't need any IE8 checks and we can always add it back in if necessary.